pkgs/by-name/vc/vcpkg/package.nix

```diff
@@ -32,6 +32,10 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   installPhase = ''
       runHook preInstall
 
+      # If we don’t turn this off, then you won’t be able to run binaries that
+      # are installed by vcpkg.
+      find triplets -name '*linux*.cmake' -exec bash -c 'echo "set(X_VCPKG_RPATH_KEEP_SYSTEM_PATHS ON)" >> "$1"' -- {} \;
```

Comment:
Sorry for asking for a change again. Is it possible to move it to the `postPatch` phase since we already patch toolchain/triplets file there? I think it's gonna be a bit cleaner like that.

Reply:
> but I would prefer if we stay explicit about which triplet we patch

Currently, there is one Linux triplet, or 21 if we include community

Do you see a reason for not patching all of them? This change seems necessary for everything on NixOS (but maybe not for Nix on another distro?)

Reply:
Yeah 21 files is a bit much, I was only looking at the official triplets. Matching all files sounds good to me then.

Reply:
> Sorry for asking for a change again.

That’s fine. I do the same thing when I’m reviewing pull requests.

> Is it possible to move it to the `postPatch` phase since we already patch toolchain/triplets file there?

Done.